<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prefix Path</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Directory</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="pathBox">
     <property name="toolTip">
      <string>Add prefix path to episode directory</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="browseButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
